### ISSUE TITLE
Space Between Times

### ISSUE THREAD

**Issue by bryantal** [BCStudentSoftwareDevTeam/cas#334]
**[Original report](https://bitbucket.org/laborstudents/cas-flask/issue/334) by Hailey Barnett (Bitbucket: [barnetth](https://bitbucket.org/barnetth), GitHub: [barnetth](https://github.com/barnetth)).**

----------------------------------------

There is a space between the times on the /BuildingManagement page that shouldn’t be there. See screenshot below for example of what I mean.

![](https://bitbucket.org/repo/y4axrX/images/2277286788-Dates.PNG)

**Comment by Sandesh-L** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [CAS #334 Space Between Times](https://trello.com/c/Ty7wHrRw/244-cas-334-space-between-times)

**Comment by tylerpar99** [comment]
Issue #334 
We were reviewing the time between spaces issue and when modifying the room under "Edit" we made a random change and saved our change. The "Last Modified" section then updated the date and time, we noticed that the date structure had been changed and the time no longer had spaces. We had made no changes at this point and after talking the issue over with Alex then determined that the issue may have been inadvertently resolved in another pull request. We believe the issue that is displayed in the screenshot was occurring due to the last modified date being prior to the fix and any new dates will display the change. 
`app/controllers/main_routes/buildingManagment.py`
